stories/foundations/Iconography.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Foundations/Iconography" />
+
+# Iconography
+
+🛠 **Work in Progress**
+
+
+Compound favours consistent iconography cross-platform. We use:
+
+- Custom icons, drawn by ourselves on the [Material 24dp icon grid and keyline template](https://m3.material.io/styles/icons/designing-icons#8754553f-29d9-4671-b163-123b8e7b8f1e)
+- [Material UI Rounded](https://fonts.google.com/icons?icon.style=Rounded) as a fallback icon set
+
+
+### Contributing an icon
+
+While we iron out initial kinks, to suggest an icon to include in Compound, designers should:
```

Comment:
I think this is extraenous. We should add a notice that this is a living document at the top of the page rather than this per section.

The expectation is for everyone to refer to this when they need to do something rather than remembering what is set in stone and what isn't

```suggestion
```